RELEASE.md

```diff
@@ -6,13 +6,14 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
-# Next Release
+# Release 6.6.1
 
 ## Major features and improvements
 
-- Skip all plugins while running Kedro Viz using the `--ignore-plugins` option. (#1544)
+- Skip all plugins while running Kedro-Viz using the `--ignore-plugins` option. (#1544)
 
 ## Bug fixes and other changes
+
```

Comment:
- Fix a bug in Kedro-viz Experiment Tracking to ensure compatibility with kedro-datasets > = 1.7.1

Reply:
Good one. That's in now.